samples/scala-value-entity-customer-registry/src/main/proto/customer/domain/customer_domain.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// tag::declarations[]
+syntax = "proto3";
+
+package customer.domain;
+
+option java_outer_classname = "CustomerDomain";
```

Comment:
The generated code in the sample is looking good, aside from that CustomerDomain is different for Scala, but I think @raboof is looking into that problem.